Sprint-1/explore/chrome.md

```diff
@@ -10,9 +10,9 @@ Let's try an example.
 In the Chrome console,
 invoke the function `alert` with an input string of `"Hello world!"`;
 
-What effect does calling the `alert` function have?
+What effect does calling the `alert` function have? - the alert function opens a message window with the message that is assigned to it.
 
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
 
-What effect does calling the `prompt` function have?
-What is the return value of `prompt`?
+What effect does calling the `prompt` function have? - the prompt give's a user an input field for the user to put an input
+What is the return value of `prompt`? - The users input
```

Comment:
If you were writing a program that uses `prompt()` to ask for an input value, how can your program tell if the user clicked "OK" or "Cancel"?

Reply:
From the research also, it seems as if I can use if else statement to console.log. For example: 

let userInput = prompt("Enter something:");

if (userInput === null) {
  console.log("User clicked Cancel");
} else {
  console.log("User clicked OK and entered:", userInput);
}